config/locales/views/circuitverse/ar.yml

```
@@ -0,0 +1,154 @@
+        item2_heading: "تحليل الدوائر المنطقية"
+        item2_text: "قم بإنشاء الدوائر تلقائيًا من جدول الحقيقة."
+        item3_heading: "اختبار الدوائر والمخططات الزمنية"
+        item3_text: "اختبر دوائرك وتحقق من سلوكها باستخدام المخططات الزمنية."
+        item4_heading: "تضمين في المدونة"
+        item4_text: "باستخدام HTML5 iFrame، يمكنك بسهولة تضمين دائرتك في المدونات والمواقع الإلكترونية."
+        item5_heading: "استخدم الدائرة الفرعية"
+        item5_text: "أنشئ دائرة فرعية مرة واحدة واستخدمها عدة مرات، مما يجعل التصميم أكثر سهولة وتنظيمًا."
+        item6_heading: "ناقل متعدد البتات والمكونات"
+        item6_text: "يدعم CircuitVerse الأسلاك متعددة البتات، مما يجعل تصميم الدوائر أكثر سهولة وسرعة وتنظيمًا."
+    featured_examples:
+        main_heading: "أمثلة مميزة"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)